common/eth2_testnet_config/built_in_testnet_configs/mainnet/deploy_block.txt

```diff
@@ -0,0 +1 @@
+11184524
```

Comment:
I'm not sure how important it is for this to be accurate, but wasn't the deposit contract deployed at 11052984?

https://etherscan.io/tx/0xe75fb554e433e03763a1560646ee22dcb74e5274b34c5ad644e7c0f619a7e1d0

Reply:
Yeah good pickup. I've set it to the block of the first deposit, since that's all we care about. It's a little hacky, perhaps we can rename this value at some point.

Reply:
Ah ok, that's fine then.